internal/database/orgs_test.go

```diff
@@ -99,6 +99,59 @@ func TestOrgs_Delete(t *testing.T) {
 	}
 }
 
+func TestOrgs_HardDelete(t *testing.T) {
+	t.Parallel()
+	db := dbtest.NewDB(t)
+	ctx := context.Background()
+
+	displayName := "org1"
+	org, err := Orgs(db).Create(ctx, "org1", &displayName)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
nit: We have this test helper
```suggestion
	require.NoError(t, err)
```